Jenkinsfile

```diff
@@ -9,163 +9,896 @@
 *                                                                                 *
 */
 
-@Library('shared-pipelines@v1.0.2') import org.zowe.pipelines.nodejs.NodeJSPipeline
+// The following property need to be set for the HTML report @TODO figure out how to get this nicely on jenkins
+//System.setProperty("hudson.model.DirectoryBrowserSupport.CSP", "")
```

Comment:
It kind of looks like you accidentally reverted the Jenkinsfile back to the old one, right @zFernand0 ?

Reply:
Fixed